var/spack/repos/builtin/packages/tar/package.py

```diff
@@ -11,15 +11,23 @@ class Tar(AutotoolsPackage):
     other kinds of manipulation."""
 
     homepage = "https://www.gnu.org/software/tar/"
-    url      = "https://ftpmirror.gnu.org/tar/tar-1.29.tar.gz"
+    url      = "https://ftpmirror.gnu.org/tar/tar-1.32.tar.gz"
 
-    version('1.31', 'b471be6cb68fd13c4878297d856aebd50551646f4e3074906b1a74549c40d5a2')
+    version('1.32', sha256='b59549594d91d84ee00c99cf2541a3330fed3a42c440503326dab767f2fbb96c')
+    version('1.31', sha256='b471be6cb68fd13c4878297d856aebd50551646f4e3074906b1a74549c40d5a2')
     version('1.30', 'e0c5ed59e4dd33d765d6c90caadd3c73')
-    version('1.29', 'cae466e6e58c7292355e7080248f244db3a4cf755f33f4fa25ca7f9a7ed09af0')
+    version('1.29', sha256='cae466e6e58c7292355e7080248f244db3a4cf755f33f4fa25ca7f9a7ed09af0')
     version('1.28', '6ea3dbea1f2b0409b234048e021a9fd7')
 
+    depends_on('libiconv')
```

Comment:
I'm not sure if this dependency is actually required, but there doesn't seem to be a way to disable it. Even if you add `--without-libiconv-prefix`, the build still picks up the system library and links to it. There is also an optional `gettext` dependency, but I don't have that on my system, so it doesn't get sucked in. Should we add `gettext` as well? Or what?